File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_backend/dto/stock/movement_details/RiceVarietyStockResponseDTO.java
package com.example.fieldsync_inventory_backend.dto.stock.movement_details;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RiceVarietyStockResponseDTO {
    private Integer riceVarietyId;
    private String riceVarietyName;
    private String riceVarietyImageUrl;
    private BigDecimal totalQuantity;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/stock_movement_details/RiceVarietyStockResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.stock_movement_details

data class RiceVarietyStockResponseDto(
    val riceVarietyId: Long,
    val riceVarietyName: String,
    val riceVarietyImageUrl: String?,
    val totalQuantity: Double
)
